src/main.py

```diff
@@ -9,12 +11,19 @@
     Multiple_PV_Actual,
     PV_Sites,
     PVSiteAPIStatus,
+    Site_Forecast_Values,
 )
+from utils import make_fake_intensity
+import pandas as pd
 
 app = FastAPI()
 
 version = "0.0.6"
 
+fake_site_uuid = "b97f68cd-50e0-49bb-a850-108d4a9f7b7e"
+
+now = pd.Timestamp(datetime.now(timezone.utc)).ceil("5T")
```

Comment:
unfortunately you need to move this inside the function, as here it is only called once and the time will soon become out of date

Reply:
good point. moved it out of the function so the code wouldn't be so repetitive. but now it's ineffective....